Directory.Build.props

```diff
@@ -444,6 +444,13 @@
   <!-- Import it at the end of the props file to override the OutputPath for reference assemblies and use common directory props -->
   <Import Project="$(MSBuildThisFileDirectory)eng\ReferenceAssemblies.props" />
 
+  <PropertyGroup Condition="'$(IsTestProject)' != 'true' AND '$(IsReferenceAssembly)' !='true'">
```

Comment:
Could you change this condition to `Condition="'$(IsSourceProject)' == 'true'"` -- IsSourceProject is a new property we added: https://github.com/dotnet/corefx/blob/24db8490eb6bdeeb2c4353d5b7f9b786546cfa2b/Directory.Build.props#L18